components/dashboard/src/teams/TeamUsage.tsx

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { useContext, useEffect, useState } from "react";
+import { Redirect, useLocation } from "react-router";
+import { PageWithSubMenu } from "../components/PageWithSubMenu";
+import { getCurrentTeam, TeamsContext } from "./teams-context";
+import { getTeamSettingsMenu } from "./TeamSettings";
+import { PaymentContext } from "../payment-context";
+import { getGitpodService } from "../service/service";
+import { BillableSession } from "@gitpod/gitpod-protocol/lib/usage";
+import { Item, ItemField, ItemsList } from "../components/ItemsList";
+import moment from "moment";
+import Property from "../admin/Property";
+import Arrow from "../components/Arrow";
+
+function TeamUsage() {
+    const { teams } = useContext(TeamsContext);
+    const { showPaymentUI, showUsageBasedUI } = useContext(PaymentContext);
+    const location = useLocation();
+    const team = getCurrentTeam(location, teams);
+    const [billedUsage, setBilledUsage] = useState<BillableSession[]>([]);
+
+    useEffect(() => {
+        if (!team) {
+            return;
+        }
+        (async () => {
+            const billedUsageResult = await getGitpodService().server.getBilledUsage("some-attribution-id");
+            setBilledUsage(billedUsageResult);
+        })();
+    }, [team]);
+
+    if (!showUsageBasedUI) {
+        return <Redirect to="/" />;
+    }
+
+    const getType = (type: string) => {
+        if (type === "regular") {
+            return "Workspace";
+        }
+        return "Prebuild";
```

Comment:
`type` is typed as `WorkspaceType` in `gitpod-protocol` but here it's just a `string`. Can we make this type more precise here too?

Reply:
Thanks for the catch 💪🏼